pkg/vpcmodel/grouping.go

```diff
@@ -23,6 +23,7 @@ type groupedCommonProperties struct {
 	conn     *common.ConnectionSet
 	connDiff *connectionDiff
 	rules    *rulesConnection
+	router   RoutingResource
```

Comment:
can the fields `rules` and `router` be replaced by one field `details` of type `srcDstDetails` ? (both are related to explanations grouping..)

Reply:
done

Reply:
why do we need a new type `explainDetails` and cannot just use the existing type `srcDstDetails` ?

Reply:
Because we will be using only 2/7 of `srcDstDetails `